### ISSUE TITLE
Manchester Township Boundaries

### ISSUE THREAD

**Issue by missinglink** [whosonfirst-data/whosonfirst-data#2158]
About a year ago I wrote the Rochdale Council about getting polygon geometries for [Heywood](https://spelunker.whosonfirst.org/id/1360791041/) and surrounding townships (which we currently only have Point geometries for)

They actually wrote back finally and gave me the shapes, when I requested the license they replied that there isn't one 🤷 is this something we can accept?

<img width="835" alt="Screenshot 2023-09-05 at 15 03 35" src="https://github.com/whosonfirst-data/whosonfirst-data/assets/738069/7566d94e-1eb8-42dc-bd04-05a4789ad781">

![Updated Township Boundaries](https://github.com/whosonfirst-data/whosonfirst-data/assets/738069/3ff754aa-9029-4e45-bfc7-5fbce4707990)

### File Attached Below

[All_Townships.zip](https://github.com/whosonfirst-data/whosonfirst-data/files/12523904/All_Townships.zip)

<img width="191" alt="Screenshot 2023-09-05 at 15 05 27" src="https://github.com/whosonfirst-data/whosonfirst-data/assets/738069/00772134-4828-4b2e-8e69-639e7fec5bc8">

**Comment by nvkelso** [comment]
That looks like copyrighted Ordinance Survey data to me? But perhaps "townships" are part of their open data program? Perhaps buried in here? https://www.ordnancesurvey.co.uk/products/boundary-line

**Comment by nvkelso** [comment]
@missinglink I don't think we can use this until we have more information. It doesn't look like an open data product to me. Can you write back to them asking about copyright on this data and if there is a general source for this type of data across all of the UK?

This problem in Manchester area is somewhat common in the UK in WOF, in that we have detailed `localadmin` outside of the `locality`, but inside `locality` we don't have any more fidelity (except in London and couple other really big cities with curated neighbourhoods). So it would be nice to solve.

Reviewing the UK's OS open boundary line product (big honking geopackage), this is the closest match I see: 

<img width="1219" alt="image" src="https://github.com/whosonfirst-data/whosonfirst-data/assets/853051/c55738eb-3354-479a-a967-a1fa5e16789c">

Those are `district_borough_unitary_ward` subdivisions of the unitary authorities, in this case for the Rochdale District (B) - CIVIL ADMINISTRATION AREA. Maybe... maybe we could squint and call these WOF `borough placetype?

I do see similar names, but even dissolving on those you wouldn't get the handful of townships in the map the local government provided you.

There's another UK OS open data for Built Up Areas: https://osdatahub.os.uk/downloads/open/BuiltUpAreas

But that is just dense urban and suburban coverage, and it's not cut up by "townships":

<img width="1216" alt="image" src="https://github.com/whosonfirst-data/whosonfirst-data/assets/853051/43880466-da07-455f-8ec6-6a5c055eea84">

Paging @vicchi do you know what OS data source has "township" divisions of a locality like in the initial (top) description of this issue? Where Rochdale (near Machnester) is composed of Rochdale South, Rochdale North, Pennines, Heywood, and Middletown as "townships"?
